packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -92,7 +92,7 @@ export class LayeredRuntimeResolverCreator {
         const deltaQuery = getDeltaQuery(resolverElement.graphqlType.name)
 
         resolvers.Query[deltaQuery] = async (parent: any, args: any, context: any) => {
-          const res = await this.services[modelName].findBy({ updatedAt:{ gt: args.lastSync }}, undefined, undefined, context);
+          const res = await this.services[modelName].findBy({ updatedAt:{ gt: args.lastSync }, _deleted: { in: [true, false]}}, undefined, undefined, context);
```

Comment:
This is going to affect all datasources sadly we cannot do that because it will break non DataSync graphback

Reply:
You probably mean datasource or DataSync service

Reply:
Yes about this, I cannot find an optimal way to add the delta resolver without doing it in layered runtime resolver

Reply:
Can we just extend this class and add deltas there?

Reply:
Also I meant that deleted should be specified on the datasource level or service

Reply:
> Also I meant that _deleted should be specified on the datasource level or service

+1 with this.